### ISSUE TITLE
Higher Resolution

### ISSUE THREAD

**Issue by Papermanzero** [libretro-mirrors/beetle-saturn-libretro#18]
Just like PSX games, Saturn games run on low resolution, which makes them very pixelated.
An option to increase the resolution would be a feature to improve image quality.

**Comment by inactive123** [comment]
It might be a bit too early to add this yet though as Mednafen Saturn is still going to receive heavy alterations by Ryphecha in the upcoming months. Maybe when work has stabilized will it be practical to do a hard fork, and maybe we should make two repos then, a hard forked one and one that stays close to upstream.

**Comment by Papermanzero** [comment]
Fully agree and fully right.
Just wanted to add the request.

**Comment by Papermanzero** [comment]
Mednafen added further improvements for saturn emulation.
Does anyone know if someone is trying to increase the internal resolution to improve the image quality?

**Comment by hiddenasbestos** [comment]
FWIW I don't have plans to work on these sort of enhancements. My personal preference is accurate and authentic recreations of the original games, warts and all. However if someone else did want to put the time into that I'm sure it would be welcomed by a lot of people (and I'd probably give it a try :) )

**Comment by eeee87** [comment]
any updates on this feature request?

**Comment by ExxistanceDC** [comment]
Would love to see this enhancement come to fruition. Does anyone if anyone is exploring it?

Yaba Sanshiro has it, but the general emulation status of that emulator isn't anywhere near as solid as Beetle Saturn.

**Comment by Papermanzero** [comment]
Agree. I like beetle saturn also more than yaba sanshiro or even kronos. However someone needs to start with a hardware renderer like in beetle psx.

**Comment by eadmaster** [comment]
> However someone needs to start with a hardware renderer like in beetle psx.

Even a software renderer @ 2x would be nice to have.
